src/pathways/pathway/pathways.service.ts

```
@@ -1202,7 +1204,8 @@ export class PathwaysService {
           updatePathwayDto.subtype !== undefined
             ? updatePathwayDto.subtype
             : existingPathway.subtype;
-        if (effectiveSubtype) {
+        const effectiveIsActive = updatePathwayDto.is_active ?? existingPathway.is_active;
```

ESLint

[error] Insert `⏎·········`

(prettier/prettier)